README.md

```diff
@@ -24,7 +24,7 @@ Please see the [contribution guide](http://definitelytyped.org/guides/contributi
 
 * Directly from the GitHub repos
 * [NuGet packages](http://nuget.org/packages?q=DefinitelyTyped)
-* [TypeScript Definition manager](https://github.com/DefinitelyTyped/tsd)
```

Comment:
I believe this is fine for now. we need to update the documentation to reflect the information in https://blogs.msdn.microsoft.com/typescript/2016/06/15/the-future-of-declaration-files/